### DISCUSSION TITLE
[A-L] (2023/24) Foglio 10 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#330] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-21 105431](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/6fd6e7b7-2af4-4564-b591-11d9437056a1)

**Comment by daromacarlo** [comment]
[foglio 10_annotato-3-5.pdf](https://github.com/sapienzastudentsnetwork/algebra2324/files/13431521/foglio.10_annotato-3-5.pdf)

**Comment by Elia-Belli** [comment]
- **(5.1)** Dobbiamo verificare che:
  - La somma di due matrici simmetriche è anch'essa simmetrica: siano $A=(a_{ij})$ e $B=(b_{ij})\in \mathcal{S}\_{nn}(\mathbb{R})$
  nella somma l'elemento $a_{ij}$ viene sommato all'elemento $b_{ij}$ e posto nella casella $ij$ della matrice somma $C=(c_{ij})$,
  sappiamo che $a_{ij}=a_{ji}$ e $b_{ij}=b_{ji}\Rightarrow c_{ij}=a_{ij}+b_{ij}=a_{ji}+b_{ji}=c_{ji}\Rightarrow C=(c_{ij})$ è simmetrica.
  - Il prodotto scalare restituisce una matrice simmetrica: moltiplicando tutti gli elementi di una matrice simmetrica per uno scalare $\lambda$ la matrice si mantiene simmetrica infatti $\lambda a_{ij}=\lambda a_{ji}\ \forall i,j$
  $\Rightarrow \mathcal{S}\_{nn}(\mathbb{R})\leq M_{nn}(\mathbb{R})$

- **(5.2)** Analogamente a prima verifichiamo che:
  - La somma di due matrici anti-simmetriche è anch'essa anti-simmetrica: siano $A=(a_{ij})$ e $B=(b_{ij})\in \mathcal{A}\_{nn}(\mathbb{R})$, sappiamo che $a\_{ij}=-a\_{ji}$ e $b_{ij}=-b_{ji}\Rightarrow c_{ij}=a_{ij}+b_{ij}=-a_{ji}-b_{ji}=-(a_{ji}+b_{ji})=-c_{ij}$
  $\Rightarrow C=(c_{ij})$ è anti-simmetrica.
  - Il prodotto scalare restituisce una matrice anti-simmetrica: infatti è evidente che $\lambda a_{ij}=-\lambda a_{ji}\ \forall i,j$
  $\Rightarrow \mathcal{A}\_{nn}(\mathbb{R}) \leq M_{nn}(\mathbb{R})$

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/e0d635da-1bbe-4c38-87c3-f88a669b7de8)

**Comment by CiottoloMaggico** [comment]
Per le antisimmetriche è identica
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/efb39195-2df9-423b-90e4-86212e69e226)
